### PR TITLE
Setup Jekyll and port over the legacy static site

### LINKED ISSUE #2: Setup and configure Jekyll
This site will be powered by [Jekyll](https://jekyllrb.com/). Design and content is still being planned, but I will go ahead and get Jekyll installed and configured in this repo. I will also move the current static page into a Jekyll template.

### PR DESCRIPTION
**Pull request by chrisvogt**
This PR resolves #2 by installing Jekyll and porting over the legacy static page to the Liquid template engine. The project readme has been updated with instructions on installing and running this project locally.

Live Preview: https://chrisvogt.github.io/godaddy.github.io/

The markup was modified slightly so that it will pass HTML5 validation without warnings or errors:

screen shot 2017-12-15 at 11 41 55 am

#### Screenshot

screen shot 2017-12-15 at 12 06 19 pm

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+  <head>
```
Comment (gergelyke) [inline _includes/head.html]: maybe it is just github, but the indent seems to be off

File: _includes/cards.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<section style="margin-top: 1em;">
+{% for project in include.projects %}
```
Comment (gergelyke) [inline _includes/cards.html]: can we add indent here?

File: _includes/head.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <link rel="stylesheet" href="https://img1.wsimg.com/ux/1.3.50-brand/css/uxcore.min.css" media="all">
+    <link rel="stylesheet" href="https://img1.wsimg.com/wrhs/d395a75515b2ebef0ec1d145e6d9ee97/appheader.min.css" media="all">
+
+    <style>
+      body {
+        background: #e8e8e8;
+      }
+    </style>
+  </head>
+
```
Comment (gergelyke) [inline _includes/head.html]: we only need one empty line, right?

Comment (chrisvogt) [reply _includes/head.html]: Oops, I forgot to set EOL characters in my IDE and after correcting that added too many here. :)

File: _includes/head.html (hunk 1 above)
Comment (chrisvogt) [reply _includes/head.html]: Not GitHub, you are correct. Fixing!

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Site settings
+name: GoDaddy OSS
+email: opensource@godaddy.com
+description: > # this means to ignore newlines until "baseurl:"
```
Comment (tarkatronic) [inline _config.yml]: Since there is no `baseurl:` key here, should this comment simply say `until the next key`?

File: _includes/head.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<head>
+  <meta charset="utf-8">
+  
+  <title>{% if page.title %} {{ page.title }} | {% endif %} {{ site.name }}</title>
```
Comment (tarkatronic) [inline _includes/head.html]: I don't know if it's intended, but there would be double spacing between a page title and the site name. Super minor, but... slightly inconsistent.

File: _includes/scripts.html [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-37178807-20"></script>
```
Comment (tarkatronic) [inline _includes/scripts.html]: Should the `id` here be using `{{ site.analytics_id }}`?

**Pull request merged**